### ISSUE TITLE
Add foo licensing compatibility

### ISSUE THREAD

**Issue by Shelob9** [CalderaWP/edd-licensing#1]
The difference between the two should be invisible to users, we figure out where the got the code than make the HTTP request to foo or cwp licensing accordingly. I totally forgot to use the EDD software license code template plugin on our site, but will add it. I figure the easiest thing to do is use the code template to force a predictable pattern or prefix on our licenses that we can check.

**Comment by DavidCramer** [comment]
yes. using the EDD template will work for it. prefix like CWP-**\*
